### PR TITLE
rfe: add cloud-init parameter to kubevirt_vmrs

### LINKED ISSUE #36: Add cloud-init to VMRS module
Cloud-init data can be used on VirtualMachineTemplate object and currently it's missing from the VMRS module.

### PR DESCRIPTION
**Pull request by tripledes**
* Added cloudinit parameter to the DOC section.
* Added cloudinit parameter to module argument spec as non-required.
* Added base64 import required to encode cloud-init data.

Fixes #36

### CODE REVIEW

File: library/kubevirt_vmrs.py [hunk 1]
```diff
@@ -59,6 +59,11 @@
             - "Local YAML file to use as a source to define the VM ReplicaSet.
                It overrides all parameters."
         required: false
+    cloudinit:
+        description:
+            - "String containing cloudInit information to pass to the VM.
```
Comment (alexxa) [inline library/kubevirt_vmrs.py]: `VM ReplicaSet` maybe?

Comment (tripledes) [reply library/kubevirt_vmrs.py]: Fixed, thanks!

**Comment by alexxa** [comment]
ops, there is a conflict. @tripledes can you update this PR please?

**Comment by tripledes** [comment]
@alexxa rebased.

**Pull request merged**